src/components/EventSchedule/EventSchedule.scss

```diff
@@ -1,9 +1,24 @@
+@import "../../styles/variables.scss";
+
+.container {
+    width: $fullWidth;
+    display: flex;
+    justify-content: center;
+    padding-top: 60px;
+}
+
+.eventsBlock {
+    width: $fullWidth;
+    display: flex;
+    justify-content: center;
```

Comment:
use mixin

Reply:
We don't have mixin just for display and j-c.